.github/workflows/ci.yml

```diff
@@ -42,7 +42,6 @@ jobs:
       image: dynawo/dynawo-ci:latest
       options: --user root
     env:
-      DYNAWO_BUILD_TYPE: TestCoverage
```

Comment:
DYNAWO_BUILD_TYPE is mandatory, should be Debug. Then DYNAWO_BUILD_TESTS_COVERAGE should be define to ON